tests/js/client/dump/dump-test.inc

```diff
@@ -1563,6 +1563,38 @@
         assertEqual(graph.orphanCollections.length, 0);
       },
 
+      testSmartGraph3_11_compat: function () {
+      return;
```

Comment:
This line looks like an unintentional change.
```suggestion
```

Reply:
😨 oh noe this is an oversight on my end, while I was trying to figure out when the test is run.
Very good catch this disabled the test!!!